src/newmatrix.cpp

```diff
@@ -1798,6 +1855,10 @@ ACMD(do_logoff)
         send_to_char("Your hitcher has disconnected.\r\n", ch);
         temp->persona->decker->hitcher = NULL;
       }
+     // Clear the deck if this is an otaku
+    if (PERSONA->decker && PERSONA->decker->deck && PERSONA->decker->deck->obj_flags.extra_flags.IsSet(ITEM_EXTRA_OTAKU_RESONANCE)) {
```

Comment:
This is being called from the case `if (!PERSONA)`, so operating on `PERSONA->decker` will cause undefined behavior and/or a segfault.